content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -104,12 +105,12 @@ while (true) {
    
 ### Methods used in ServerSocket
```

Comment:
Change:

> ### Methods used in ServerSocket

To:

> #### Methods used in ServerSocket

Fixing heading structure since this is under: **Server-side Programming**